design_notes/prng.md

```diff
@@ -96,8 +96,11 @@ def main():
 ```
 
 Some points to notice:
-- there is no sequential dependence between the calls to bar() and baz() and they can be evaluated in either order without affecting the value of the result, which solves the remaining performance goals (#5, #6),
```

Comment:
Nit: it would probably be better to keep this document as pure Markdown, without using HTML.

Markdown doesn't seem to support lettered lists, but it does support numbered lists and a numbered list would work here I think?

Reply:
Done - thanks!